### PR TITLE
[16.0][MIG] stock_product_set: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Pending from:

- [x] https://github.com/OCA/product-attribute/pull/1548

cc @Tecnativa TT48100

@victoralmau @carolinafernandez-tecnativa please review

### CODE REVIEW

File: stock_product_set/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Add product sets in pickings",
+    "version": "16.0.1.0.0",
+    "category": "Warehouse",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["stock", "sale_product_set"],
```
Comment (victoralmau) [inline stock_product_set/__manifest__.py]:
```suggestion
    "depends": ["stock", "product_set"],
```

Comment (victoralmau) [reply stock_product_set/__manifest__.py]: `sale_product_set` module v15 was splitted in v16 (https://github.com/OCA/product-attribute/pull/1384). 
This module should now depend only on `product_set`. Remove everything related to `sale` (no longer needed).

**Review by victoralmau — CHANGES_REQUESTED** [review]

File: stock_product_set/__manifest__.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply stock_product_set/__manifest__.py]: The wizard that opens the "Add set" button in planned transfer is defined in the module sale_product_set so it must still depend on sale_product_set even if the logic is in product_set.

https://github.com/OCA/sale-workflow/blob/16.0/sale_product_set/wizard/product_set_add.py#L9

Comment (pilarvargas-tecnativa) [reply stock_product_set/__manifest__.py]: Dependency removed

**Comment by pilarvargas-tecnativa** [comment]
@victoralmau Changes made to eliminate dependency at https://github.com/OCA/product-attribute/pull/1548

**Review by victoralmau — APPROVED** [review]
Code and functional review OK.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM, Thanks :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**